pymatviz/uncertainty.py

```diff
@@ -84,7 +92,7 @@ def qq_gaussian(
         ax.fill_between(
             exp_proportions, y1=obs_proportions, y2=exp_proportions, alpha=0.2
         )
-        miscal_area = np.trapz(  # noqa: NPY201
```

Comment:
Looks like this line is not covered by unit test, this should fail in NumPy 2.x because `np.trapz` is removed.